specutils/analysis/flux.py

```diff
@@ -71,6 +84,10 @@ def equivalent_width(spectrum, continuum=1, regions=None):
         will be assumed, otherwise units are required and must be the same as
         the ``spectrum.flux``.
 
+    mask_interpolation : `~specutils.manipulation.LinearInterpolatedResampler`
+        Interpolator class used to fill up the gaps in the spectrum's flux
+        array, when the spectrum mask is not None.
```

Comment:
Just some clarification. It was not immediately obvious why we'd be interpolating. But it's because there may be a excise operation that changes the shape of the flux/spectral axis without changing the shape of mask (and thus the mask would not be able to be applied in such cases).

```suggestion
    mask_interpolation : `~specutils.manipulation.LinearInterpolatedResampler`
        Interpolator class used to fill up the gaps in the spectrum's flux
        array after an excise operation to ensure the mask shape can always be applied when 
        the spectrum mask is not None.
```